core/core_modules/disable.js

```diff
@@ -1,24 +1,105 @@
-var disabled = false;
+var prevTimeStamp;
+var counter = 0;
+var commands = [
+  'disable',
+  'counter',
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 space characters but found 2.](https://www.codacy.com/app/theprojectshop/Kassy/file/2556388143/issues/source?bid=3282987&fileBranchId=3282995#l5)